catalogs/data/bedrock/bedrock-stack.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: deployments.plural.sh/v1alpha1
+kind: InfrastructureStack
+metadata:
+  name: bedrock-{{ context.cluster }}
```

Comment:
i still think this would be better w/o the cluster paramterized in this name, something like just `bedrock-proxy`.  It's not going to be redeployed that much.